app/views/posts/index.html.erb

```diff
@@ -1 +1,25 @@
-<h1> Lists of Posts for a specific User </h1>
\ No newline at end of file
+<section>
+  <h1> Lists of Posts for a specific User </h1>
+  <div>
+    <img src=<%=@author.photo%>/>
+    <h2><%=@author.name%></h2>
+    <span> <%=@author.posts_count%> </span>
+  </div>
+  <ul>
+   <%@author.posts.each do |post|%>
+    <li> 
+      <div>
+        <h3><%=post.title %></h3>
+        <p><%=post.text%></p>
+        <div>
+          <span> <%=post.comments_count%> </span>
+           <span> <%=post.likes_count%> </span>
+        </div>
+        <%post.comments.each do |comment|%>
+          <li><p> <span><%=comment.author.name %> :</span> <%=comment.text%>  </p></li>
+        <%end%>
+      </div>
+    </li>
+    <%end%>
+  </ul>
```

Comment:
- [ ] When you open a user post you should see only 3recent posts and a `link` that leads you to a page that has all posts please work on that, currently I am seeing all posts

Reply:
Hi @DeliceLydia , the requirements suggests that we should display recent posts for /user.id path which links to users#how page, and for posts/index.html we are suppose to display all posts and 5 recent comments which i have implemented.